lms/djangoapps/grades/migrations/0005_multiple_course_flags.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import xmodule_django.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('grades', '0004_visibleblocks_course_id'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='coursepersistentgradesflag',
+            name='course_id',
+            field=xmodule_django.models.CourseKeyField(max_length=255, db_index=True),
+        ),
+    ]
```

Comment:
@efischer19 I was out, but to confirm (on this merged PR) dropping this index on a table this small is fine.